setup.py

```diff
@@ -36,7 +36,10 @@
     use_scm_version=True,
     setup_requires=["setuptools_scm"],
     python_requires=">=3.7",
-    install_requires=["django>=3.2", "django-ipware>=3", "setuptools"],
+    install_requires=["django>=3.2", "setuptools"],
+    extras_require={
+        "ipware": "django-ipware>=3",
```

Comment:
Wondering if this should be `ipware`,  `django-ipware` or both in the `extras` section.

Reply:
It seems to me that `ipware` is enough, this is more brief, especially in the context of django it seems clear to me that `django-ipware` is meant

Reply:
Yeah let's keep it just `ipware`. It's explicit enough and it'll be easy to add more specifiers later if needed 👍